circuits-circom/hdfc_send.circom

```diff
@@ -0,0 +1,136 @@
+pragma circom 2.1.5;
+
+include "circomlib/circuits/poseidon.circom";
+// include "./utils/email_verifier.circom";
+include "@zk-email/circuits/helpers/extract.circom";
+include "./stubs/email-verifier.circom";
+include "./utils/ceil.circom";
+include "./utils/email_nullifier.circom";
+include "./utils/extract.circom";
+include "./utils/hash_sign_gen_rand.circom";
+include "./helpers/hdfc_helpers.circom";
+include "./regexes/from_regex.circom";
+include "./regexes/hdfc/hdfc_amount.circom";
+include "./regexes/hdfc/hdfc_date.circom";
+include "./regexes/hdfc/hdfc_upi_subject.circom";
+
+template HdfcSendEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
+    assert(n * k > 1024); // constraints for 1024 bit RSA
+
+    // Rounded to the nearest multiple of pack_size for extra room in case of change of constants
+    var max_email_amount_len = 8; // Allowing max 4 fig amount+ one comma  + one decimal point + 2 decimal places. e.g. 2,500.00
```

Comment:
What is the max limit for UPI? Currently this supports 9,999.99 INR as the max which isn't that high